packages/berlin/src/components/cycles/Cycles.tsx

```diff
@@ -34,7 +38,11 @@ function Cycles({ cycles, errorMessage, eventId }: CyclesProps) {
           </article>
         ))
       ) : (
-        <Body>{errorMessage}</Body>
+        <section className="flex w-full flex-col items-center gap-4 pt-12">
+          <CalendarX2 width={64} height={64} />
+          <Body>{errorMessage}</Body>
+          <Button onClick={() => setActiveTab(tab)}>{tab} questions</Button>
```

Comment:
move this to the parent component and add an errorOnClick method here. the main reason is to not have a tab prop that can become confusing as it really always means to "what tab should i go to if i can not be here".